bundles/org.openhab.binding.boschshc/pom.xml

```diff
@@ -14,23 +14,27 @@
 
   <name>openHAB Add-ons :: Bundles :: Bosch Smart Home Binding</name>
 
+  <properties>
+    <version.bouncycastle>1.78.1</version.bouncycastle>
```

Comment:
@jlaur @lsiepel @morph166955 do you know whether this version is available in the runtime (of all three bundles)? Where can I find out which versions are currently shipped?

Reply:
I don't think we bundle bouncycastle in the main distro (although we probably should).  AndroidTV was just moved to 1.78.  Not sure the differences to 1.78 vs 1.78.1 off hand.

Reply:
What is the best way to proceed here? Who can we contact about bundling it in the main distro?

And in case it is not bundled: should we go for `1.78.1` or should we align the version with `androidtv` (`1.78`)? For reference, here is the `androidtv` PR: https://github.com/openhab/openhab-addons/pull/16781/files

Reply:
I think #3302 is related? Perhaps @holgerfriedrich can advise?

Reply:
I think we will never mange to keep all add-ons and OH core in sync. So I am fine with using the latest release here.
For core, I decided to stay in sync with Karaf in openhab/openhab-core#4230, because I cannot clearly test the implications of mixed versions.

We did not manage to proceed with openhab/openhab-core#3302 yet.  IIRC, dependency to Bouncy Castle comes from Karaf only.

Reply:
I would concur with keeping it at the Karaf version.  That said, it may not hurt to add bouncy to addons directly instead of having it on a per-addon basis.  Definitely positives/negatives of doing that but at least it's all uniform then across bindings.

Reply:
The only problem with aligning with core version 1.77 is that it doesn't fix all the security issues - 1.78 is needed for that. So if core needs 1.77 to be aligned with karaf, but addons doesn't need to be aligned with core, perhaps we should upgrade both boschshc and androidtv to 1.78.1? See also #16781.

Reply:
I think keeping all add-ons in sync with OH core would be a burden for the next Karaf upgrades. We cannot judge which add-ons might break during the upgrade. BC usage might not be covered by the tests. This is to be accepted by add-on owners.
Another fact is that OH core is typically late with upgrading, due to dependencies to upstream packages. Currently, core is still at BC 1.77, known for a set of CVEs.

Hopefully we will proceed with the crypto features in core at some time and get a better basis for the add-ons.

Reply:
Thank you for your replies. From what I gathered, we could do the following:

* merge this PR as proposed
* optionally bump the BC dependencies in `androidtv` to `1.78.1` (although I'm not sure what the exact differences are between `1.78` and `1.78.1`)
* continue with general enhancements like https://github.com/openhab/openhab-core/pull/3302
* consider whether it makes sense to keep addons and core dependencies in sync

From my perspective, the primary goal should be to fix the security vulnerabilities first, then consider potential enhancements after that.

Reply:
Agree, let's merge this and aslo update androidtv to same version.